frontend/packages/dev-console/src/components/import/GitImportForm.tsx

```diff
@@ -27,16 +28,23 @@ const GitImportForm: React.FC<FormikProps<FormikValues> & GitImportFormProps> =
   return (
     <form onSubmit={handleSubmit} data-test-id="import-git-form">
       <FormBody>
-        <GitSection buildStrategy={values.build.strategy} builderImages={builderImages} />
-        <BuilderSection image={values.image} builderImages={builderImages} />
-        <DockerSection buildStrategy={values.build.strategy} />
-        <AppSection
-          project={values.project}
-          noProjectsAvailable={projects.loaded && _.isEmpty(projects.data)}
-        />
-        <ResourceSection />
-        <PipelineSection builderImages={builderImages} />
-        <AdvancedSection values={values} />
+        <GitSection builderImages={builderImages} />
+        {values.git.validated === ValidatedOptions.success && (
```

Comment:
I think we need an option to show the rest of the form even if the url isn't validated, for eg. when an unsupported repo host is used.